resources/authgear/templates/en/web/authflowv2/__bot_protection_controller.html

```diff
@@ -1,5 +1,5 @@
 {{ if (eq $.BotProtectionProviderType "cloudflare" ) }}
-  data-controller="cloudflare-turnstile" data-cloudflare-turnstile-site-key-value="{{$.BotProtectionProviderSiteKey}}"
+  data-controller="cloudflare-turnstile" data-cloudflare-turnstile-site-key-value="{{$.BotProtectionProviderSiteKey}}" data-cloudflare-turnstile-lang-value="{{ $.ResolvedLanguageTag }}"
```

Comment:
I expect you explain how our language tags can be perfectly fit into cloudflare turnstile without any mapping.

According to the documentation, https://developers.cloudflare.com/turnstile/reference/supported-languages/ it does not support zh-HK, which we supports.

Please introduce ResolvedLanguageTagForCloudflareTurnstile, and ResolvedLanguageTagForRecaptchaV2. Please add a comment to AvailableLanguages to tell the future editor when they add a new language, they have to be aware of Cloudflare turnstile and Recaptchav2.